Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Installation.rst

```diff
@@ -20,11 +20,19 @@ which boil down to this in the simplest case:
 
 Then use `Composer`_ in a directory which will be accessible by your web server to download
 and install all packages of the Flow Base Distribution. The following command will
-clone the 3.0.0 version, include development dependencies and keep git metadata for future use:
+clone the latest stable version, include development dependencies and keep git metadata
+for future use:
 
 .. code-block:: sh
 
- composer create-project --keep-vcs neos/flow-base-distribution tutorial 3.0.0
+ composer create-project --keep-vcs neos/flow-base-distribution tutorial
+
+This will install the latest stable version of Neos. In order to install a specific version, type
+
+.. code-block:: sh
+ composer create-project --keep-vcs neos/flow-base-distribution tutorial 3.3
+
+to install the last LTS version for example.
```

Comment:
now we'll have to maintain this again and don't forget to update the version with the next LTS. Why not use a placeholder here?

Reply:
I'm always bad with placeholders, I never find them obvious enough to not make any mistakes or create questions (is this character part of the placeholder or not?). Do you have any suggestion for how this could look?

Reply:
But wasn't that the point of this PR to get rid of the fixed version string? ;)

> Do you have any suggestion for how this could look?

Maybe sth like this:

```
The following command will clone the latest stable version, include development dependencies and keep git metadata for future use into a folder `tutorial`:

    composer create-project --keep-vcs neos/flow-base-distribution tutorial

In order to install a *specific version*, type

    composer create-project --keep-vcs neos/flow-base-distribution <target-directory> <version>

And replace `<target-directory>` with the folder name to create the project in and `<version>` with the specific version to install, for example `1.2`.
See [Composer documentation](https://getcomposer.org/doc/03-cli.md#create-project) for further details.
```
?

Reply:
I couldn't have come up with this somehow :)